portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>STEP Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body class="background">
+    <h1 class="title">STEP Blog</h1>
+    <div id="top-navigation"> </div>
```

Comment:
nit: Are the spaces between the opening and closing div needed here? If not please remove! Please do the same for the other files as well!